app/models/concerns/field_populators/skill_level_fields.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+module FieldPopulators
+  module SkillLevelFields
```

Comment:
Hm, I fear this does not really fit a module. From what I remember it was implemented as a separate class (and not adding a constant to the `ApplicationDraft` model, since it also has some **bahavior** (see the metaprogramming fun on the Eigenclass) and is used in 2 different places (`Mentor` views and `ApplicationDraft` views).

Given it has behavior attached to the Eigenclass, I'd say moving this into a module and including it somewhere else is *entering dangerzone* - I just would not do it because it's super hard to grasp all the potential sideeffects this can have 🔥